controller/lib/core/controller.cpp

```diff
@@ -19,65 +19,69 @@ limitations under the License.
 #include "vars.h"
 #include <math.h>
 
-// PID-tuning were chosen by following the Ziegler-Nichols method,
-// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method
-//
-// Note that Ku and Tu only seem to work with this particular sample time.
 static constexpr Duration PID_SAMPLE_PERIOD = milliseconds(10);
-static constexpr float Ku = 200;
-static constexpr Duration Tu = seconds(1.5f);
-
-// "No overshoot" settings from the Ziegler-Nichols Wikipedia page.  This
-// avoids overpressurizing the patient's lungs.
-static constexpr float Kp = 0.2f * Ku;
-static constexpr float Ki = 0.4f * Ku / Tu.seconds();
-static constexpr float Kd = Ku * Tu.seconds() / 15;
 
 static DebugFloat dbg_setpoint("setpoint", "Setpoint pressure, kPa");
 
+static DebugFloat dbg_kp("kp", "Proportional gain for main loop", 3.5);
+static DebugFloat dbg_ki("ki", "Integral gain for main loop", 1.0);
+static DebugFloat dbg_kd("kd", "Derivitive gain for main loop");
```

Comment:
typo: derivative

Reply:
Thanks, my spelling is attrocious [sic].